code/controllers/subsystems/jobs.dm

```diff
@@ -505,7 +505,7 @@ SUBSYSTEM_DEF(jobs)
 			H.forceMove(S.loc)
 		else
 			var/decl/spawnpoint/spawnpoint = job.get_spawnpoint(H.client)
-			H.forceMove(pick(spawnpoint.turfs))
+			H.forceMove(length(spawnpoint.spawn_turfs) ? pick(spawnpoint.spawn_turfs) : locate(1,1,1))
```

Comment:
```suggestion
			H.forceMove(DEFAULTPICK(spawnpoint.spawn_turfs, locate(1,1,1)))
```
I kind of feel like we should log if it doesn't have turfs, though. Is that done elsewhere?